roles/cluster_config/README.md

```diff
@@ -0,0 +1,31 @@
+# cluster_config
+
+Role cluster_config can be used to:
+- fully configure a new HyperCore server
+- partially reconfigure an existing HyperCore server
+
+## Requirements
+
+- NA
+
+## Role Variables
+
+See [argument_specs.yml](../../roles/cluster_config/meta/argument_specs.yml).
+
+## Limitations
+
+- NA
+
+## Dependencies
+
+- NA
```

Comment:
Should we add information about dependencies on valid DNS / internet connection somewhere in here?
OIDC for example can't be configured in case of external AD without a valid connection.
Or is this something that is self explanatory?
We could also add this information in the module if missing.

Reply:
I think this is not needed. About half of cluter configuration modules requires IP connectivity and DNS resolution (SMTP server, email alerts, syslog server, even NTP server).

This whole readme - I want to throw it away, and replace it with documentation generated from argument_spec.